python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-dashboard-config.json

```diff
@@ -0,0 +1,331 @@
+            ],
+            [
+               ".",
+               "usage_user",
+               ".",
+               ".",
+               ".",
+               "."
+            ],
+            [
+               ".",
+               "usage_idle",
+               ".",
+               ".",
+               ".",
+               "."
+            ]
+         ],
+         "region":"{region}",
+         "stat":"Average",
+         "period":60,
+         "title":"CPU Usage",
+         "yAxis":{
+            "left":{
+               "min":0,
+               "max":100
+            }
+         }
+      }
+   },
+   {
+      "type":"metric",
+      "x":0,
+      "y":12,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "metrics":[
+            [
+               "{cluster_name}-ray-CWAgent",
+               "processes_running",
+               "InstanceId",
+               "{instance_id}",
+               {
+                  "label":"Runnng",
+                  "color":"#4b74bd"
+               }
+            ],
+            [
+               ".",
+               "processes_sleeping",
+               ".",
+               ".",
+               {
+                  "label":"Sleeping",
+                  "color":"#69cbe2"
+               }
+            ],
+            [
+               ".",
+               "processes_dead",
+               ".",
+               ".",
+               {
+                  "label":"Dead",
+                  "color":"#b1d28c"
+               }
+            ],
+            [
+               ".",
+               "processes_zombies",
+               ".",
+               ".",
+               {
+                  "label":"Zombies",
+                  "color":"#78574b"
+               }
+            ]
+         ],
+         "region":"{region}",
+         "stat":"Average",
+         "period":60,
+         "title":"Process Types"
+      }
+   },
+   {
+      "type":"metric",
+      "x":12,
+      "y":12,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "metrics":[
+            [
+               "{cluster_name}-ray-CWAgent",
+               "processes_total",
+               "InstanceId",
+               "{instance_id}",
+               {
+                  "label":"Total",
+                  "color":"#69cbe2"
+               }
+            ],
+            [
+               ".",
+               "processes_total_threads",
+               ".",
+               ".",
+               {
+                  "label":"Total Threads",
+                  "color":"#f1b3aa"
+               }
+            ]
+         ],
+         "region":"{region}",
+         "stat":"Average",
+         "period":60,
+         "title":"Process Count"
+      }
+   },
+   {
+      "type":"metric",
+      "x":0,
+      "y":18,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "stacked":false,
+         "metrics":[
```

Comment:
When we convert this to a cluster-level search expression, I'd recommend that we cut out all the additional devices and paths that we're trying to monitor here and just monitor the root path by default (i.e. `MetricName="disk_used_percent" path="/"`)